phase/prepare_arm.go

```diff
@@ -0,0 +1,58 @@
+package phase
```

Comment:
I just do not understand what is wrong with this file now.

Vim gives me:
![image](https://user-images.githubusercontent.com/224971/129533637-2be4a4d6-3b2d-4a09-9945-d4e89f31a634.png)

`go vet` gives me:

```
# github.com/k0sproject/k0sctl/cmd
cmd/apply.go:79:5: undefined: phase.PrepareArm
# github.com/k0sproject/k0sctl/cmd
vet: cmd/apply.go:79:11: PrepareArm not declared by package phase
```

any ideas, @erdii @jnummelin @jasmingacic  ?

Reply:
hmmmmmmmmmmm, is it the `_arm` in filename? maybe.

Reply:
Yes it is.

> Files with os and architecture specific suffixes automatically follow those same constraints, e.g. name_linux.go will only build on linux, name_amd64.go will only build on amd64. This is the same as having a `//+build amd64` line at the top of the file